hypothesis-python/tests/conjecture/test_ir.py

```diff
@@ -774,6 +789,41 @@ def test_forced_nodes_are_trivial(node):
             },
             was_forced=False,
         ),
+        IRNode(
+            ir_type="integer",
+            value=1,
+            kwargs={
+                "min_value": -10,
+                "max_value": None,
+                "weights": None,
+                "shrink_towards": 1,
+            },
+            was_forced=False,
+        ),
+        IRNode(
+            ir_type="integer",
+            value=1,
+            kwargs={
+                "min_value": None,
+                "max_value": 10,
+                "weights": None,
+                "shrink_towards": 1,
+            },
+            was_forced=False,
+        ),
+        # we don't consider shrink_towards for unbounded integers.
+        # the trivial value should probably be 1 here, not 0.
```

Comment:
I'm pretty sure we should change this, though maybe in a follow-up PR.

Reply:
agree - I was hoping to hold off until ordering is on the ir though, where this will be simpler to implement. If nobody has noticed yet then it should be ok for another month?